src/contrib/persistence/Akka.Persistence.Sql.Common/Snapshot/SqlSnapshotStore.cs

```diff
@@ -123,7 +130,7 @@ private bool WaitingForInitialization(object message) => message.Match()
             })
             .With<Failure>(failure =>
             {
-                _log.Error(failure.Exception, "Error during snapshot store initialization");
+                Log.Error(failure.Exception, "Error during snapshot store initialization");
```

Comment:
This was initially calling `_log`, which may or may not be null at the time.  Changed to `Log` which will instantiate a default if `_log` is null.